notifiers/providers/dingtalk.py

```diff
@@ -0,0 +1,39 @@
+from ..core import Provider
+from ..core import Response
+from ..utils import requests
+
+
+class DingTalk(Provider):
+    """Send DingTalk notifications"""
+
+    base_url = "https://oapi.dingtalk.com/robot/send?access_token={}"
```

Comment:
best to pass `access_token` in a `params` variable to the request via `_send_notification`

Reply:
yes, but dingtalk robot api just a normal notification case, that the access_token same as webhook_url. it support some security function like keywords ident, allowed ip range, and signature hash, this commit used keywords ident function.

the keyword ident is inside msg_data object, i'll detail this later.